src/emulator/controller.ts

```diff
@@ -164,6 +188,17 @@ export async function startAll(options: any): Promise<void> {
       auto_download: true,
     };
 
+    if (exportMetadata.firestore) {
+      const importDirAbsPath = path.resolve(options.import);
+      const firestoreExportDir = path.join(importDirAbsPath, exportMetadata.firestore);
+      const exportFiles = fs.readdirSync(firestoreExportDir);
+      const metadataFileName = exportFiles.find((f) => f.endsWith("overall_export_metadata"));
```

Comment:
So for the firestore emulator we just say `export_name: foo` and it creates `foo/foo.overall_export_metadata` but I have made this more predictable now.

Reply:
Thanks! One nit: How do you feel about writing `{firestore:{metadata_file: 'foo/foo.overall_export_metadata'}}` to file instead of `firestore: 'foo'`?

Reply:
SGTM